docs/sources/setup/install/helm/install-microservices/_index.md

```diff
@@ -0,0 +1,223 @@
+It is not recommended to run microservices mode with `filesystem` storage.
+
+**Prerequisites**
+
+- Helm 3 or above. See [Installing Helm](https://helm.sh/docs/intro/install/).
+- A running Kubernetes cluster.
+- (Optional) A Memcached deployment for better query performance. For information on configuring Memcached, refer to [caching section]({{< relref "../../../../operations/caching" >}}).
+
+
+**To deploy Loki in simple scalable mode:**
+
+
+1. Add [Grafana's chart repository](https://github.com/grafana/helm-charts) to Helm:
+
+   ```bash
+   helm repo add grafana https://grafana.github.io/helm-charts
+   ```
+
+1. Update the chart repository:
+
+   ```bash
+   helm repo update
+   ```
+
+1. Configure the object storage:
+
+   - Create the configuration file `values.yaml`. The example below illustrates how to deploy Loki in test mode using MinIO as storage:
+
+     ```yaml
+        loki:
+          schemaConfig:
+            configs:
+              - from: 2024-04-01
+                store: tsdb
+                object_store: s3
+                schema: v13
+                index:
+                  prefix: loki_index_
+                  period: 24h
+          ingester:
+            chunk_encoding: snappy
+          tracing:
+            enabled: true
+          querier:
+            # Default is 4, if you have enough memory and CPU you can increase, reduce if OOMing
+            max_concurrent: 4
+
+        #gateway:
+        #  ingress:
+        #    enabled: true
+        #    hosts:
+        #      - host: FIXME
+        #        paths:
+        #          - path: /
+        #            pathType: Prefix
+
+        deploymentMode: Distributed
+
+        ingester:
+          replicas: 3
+        querier:
+          replicas: 3
+          maxUnavailable: 2
+        queryFrontend:
+          replicas: 2
+          maxUnavailable: 1
+        queryScheduler:
+          replicas: 2
+        distributor:
+          replicas: 3
+          maxUnavailable: 2
+        compactor:
+          replicas: 1
+        indexGateway:
+          replicas: 2
+          maxUnavailable: 1
+
+        bloomCompactor:
+          replicas: 0
+        bloomGateway:
+          replicas: 0
+
+        # Enable minio for storage
+        minio:
+          enabled: true
+
+        # Zero out replica counts of other deployment modes
+        backend:
+          replicas: 0
+        read:
+          replicas: 0
+        write:
+          replicas: 0
+
+        singleBinary:
+          replicas: 0 
+     ```
+
+     To configure other storage providers, refer to the [Helm Chart Reference]({{< relref "../reference" >}}).
+
+
+1. Install or upgrade the Loki deployment.
+     - To install:
+        ```bash
+       helm install --values values.yaml loki grafana/loki
+       ```
+    - To upgrade:
+       ```bash
+       helm upgrade --values values.yaml loki grafana/loki
+       ```
+       
+
+1. Verify that Loki is running:
+    ```bash
+    kubectl get pods -n loki
+    ```
```

Comment:
I think it's helpful to add example output from this command so that users know what pods they should expect to see.